### DISCUSSION TITLE
Footer information?

### DISCUSSION THREAD

**Discussion by nevenc** [chipzoller/hugo-clarity#438] [Q&A]
Hey team,

Has anyone changed footer information to include attribution to this great theme?
What's the best way to customize that without changing the theme itself?
Do I have to change the theme layouts?
How would you do it?

Thanks!

**Comment by onweru** [comment]
@nevenc,

At the moment it's possible to do so [via 🔗 ](https://github.com/chipzoller/hugo-clarity/blob/aaf6da813c6352873d8ea4a9a5ab3de8c646f138/i18n/en.toml#L4-L5). You can't do much but you can use markdown to include a link & bold text. For example 👇🏼 

```toml
[copyright]
other = "something about the [theme](#link)"
``` 

If you wanted to submit a PR to improve this, here is [where you'd start](https://github.com/chipzoller/hugo-clarity/blob/aaf6da813c6352873d8ea4a9a5ab3de8c646f138/layouts/partials/footer.html#L11)
